### ISSUE TITLE
Sending email with SMTP transport is failing on Craft 3.5.2

### ISSUE THREAD

**Issue by ockam** [putyourlightson/craft-campaign#164]
### Describe the bug
After upgrading Craft to 3.5.2, I try to send a test email from the email settings and it fails with error:
```
2020-08-14 09:02:08 [-][1][84066a4b1566e73baac7120e4e382142][error][yii\base\UnknownMethodException] yii\base\UnknownMethodException: Calling unknown method: craft\mail\transportadapters\Smtp::beforeSave() in /home3/cegepfxg/craft-flash/vendor/yiisoft/yii2/base/Component.php:300
Stack trace:
#0 /home3/cegepfxg/craft-flash/vendor/putyourlightson/craft-campaign/src/controllers/SettingsController.php(450): yii\base\Component->__call('beforeSave', Array)
#1 [internal function]: putyourlightson\campaign\controllers\SettingsController->actionSendTestEmail()
#2 /home3/cegepfxg/craft-flash/vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#3 /home3/cegepfxg/craft-flash/vendor/yiisoft/yii2/base/Controller.php(180): yii\base\InlineAction->runWithParams(Array)
#4 /home3/cegepfxg/craft-flash/vendor/craftcms/cms/src/web/Controller.php(181): yii\base\Controller->runAction('send-test-email', Array)
#5 /home3/cegepfxg/craft-flash/vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('send-test-email', Array)
#6 /home3/cegepfxg/craft-flash/vendor/craftcms/cms/src/web/Application.php(294): yii\base\Module->runAction('campaign/settin...', Array)
#7 /home3/cegepfxg/craft-flash/vendor/craftcms/cms/src/web/Application.php(597): craft\web\Application->runAction('campaign/settin...', Array)
#8 /home3/cegepfxg/craft-flash/vendor/craftcms/cms/src/web/Application.php(273): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#9 /home3/cegepfxg/craft-flash/vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#10 /home3/cegepfxg/flash/index.php(24): yii\base\Application->run()
#11 {main}
2020-08-14 09:02:08 [-][1][84066a4b1566e73baac7120e4e382142][info][application] $_GET = [
    'p' => 'admin/campaign/settings/email'
]
```

### Expected behaviour
Test email should work with SMTP.

### Versions
- Plugin version: 1.15.5
- Craft version: 3.5.2

**Comment by ockam** [comment]
After some digging, my bug may not be related to Campaign itself: I’m unable to use my SMTP credential in  Craft Mail config either. 

I can use my SMTP credentials with Telnet, so I know they work. I wonder if this can be a bug in Craft or Yii (or SwiftMailer).

**Comment by ockam** [comment]
I had an issue with my SMTP connection when using TLS (it worked before but has stopped working for a reason I ignore). Disabling encryption solved the problem.

But, I still get the error on line 450 when trying Campaign mail config. If I comment it out in the code, it works fine.

**Comment by bencroker** [comment]
Fixed in https://github.com/putyourlightson/craft-campaign/commit/768c73d2f55f9a4fd295b76ce6f197f7ba1ed026 for the next release, thanks!

**Comment by ockam** [comment]
@bencroker is it normal that 1.15.6 is _unreleased_?

**Comment by bencroker** [comment]
Yes, there is another feature in development for it. I'll aim to release it today though.

**Comment by bencroker** [comment]
Just released 1.15.6.
